src/services/serverService.ts

```diff
@@ -28,4 +28,11 @@ export class ServerService {
   public async getLatestVersion(): Promise<Either<ActionResponse, string>> {
```

Comment:
Unrelated, but I spotted this

The version we are using in the Server should not be grabbed from the github API
There is an endpoint (about uses it) that displays the version of the server we are using.
Is misleading, IMO, putting this version like current version as the latest one, and won't work for DG
IMO this call should be removed and use the server version API

Reply:
Removed in #377